Doc/library/unittest.rst

```diff
@@ -607,7 +607,7 @@ Distinguishing test iterations using subtests
 
 .. versionadded:: 3.4
 
-When some of your tests differ only by a some very small differences, for
+When some of your tests differ only by some very small differences, for
```

Comment:
Thanks for your PR, but I think that "some very small differences" is sounding odd 😕 

What about

```suggestion
When there are very small differences among your tests, for
```